db/migrate/20170720090144_add_es_mapping_for_account.rb

```diff
@@ -1,5 +1,6 @@
 class AddEsMappingForAccount < ActiveRecord::Migration
   def change
+    return
```

Comment:
if this migration is no longer needed, we can delete it

Reply:
yes not needed, will delete it